docs/user/guides/pdf-non-ascii-languages.rst

```diff
@@ -3,8 +3,6 @@ Sphinx PDFs with Unicode
 
 Sphinx offers different `LaTeX engines`_ that have better support for Unicode characters
 and non-European languages like Japanese or Chinese.
-By default Sphinx uses ``pdflatex``,
-which does not have good support for Unicode characters and may make the PDF builder fail.
```

Comment:
Can we keep "By default, Sphinx uses `latexmk <https://mg.readthedocs.io/latexmk.html>`_" or something similar?

I'm not sure how it affects Unicode characters, so seems good to delete.